tests/SpecTests/Operation.php

```diff
@@ -158,6 +158,7 @@ public static function fromCrud(stdClass $operation)
     {
         $o = new self($operation);
 
+        $o->errorExpectation = ErrorExpectation::fromCrud($operation);
```

Comment:
I see that `ResultExpectation::fromCrud()` did check for an error result, but that only changed the assertion type to `ASSERT_NOTHING`. Is this fixing an outstanding bug where the CRUD spec tests were never asserting error exceptions (since being ported over to the `MongoDB\Tests\SpecTests\FunctionalTestCase` runner?

If so: (a) oops, and (b) can we track this with a JIRA ticket?

Reply:
Yes and no. For one, the error message is more sensible when having `ErrorExpectation` in place, as it fails if an error was recorded even though none was expected. At the same time, we didn't check for the presence of an error for the two tests that throw one, so those tests were definitely incomplete. I've created [PHPLIB-482](https://jira.mongodb.org/browse/PHPLIB-482) to track this and I've also amended the commit to reference the ticket.

Reply:
That all makes sense. Thanks for catching this!